### DISCUSSION TITLE
Food Porn

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#304] [Imagens]
Post de referência: https://www.vish.blog.br/artigos/food-porn/

**Comment by brunoaugusto** [comment]
Uma vez eu vi um vídeo dessa tal de Boki da última imagem comendo ovos cozidos e cara, a mina é um saco sem fundo! Tipo, eu até sei que alguns desses *meokbangers* cospem/vomitam a comida depois de comer — todo "reality show" tem disso — mas se ela realmente comer tudo ela é tipo uma versão *live action* da Lendária Rainha Glutona :stuck\_out\_tongue\_closed\_eyes:

**Comment by one-eyed-willy** [comment]
![null](<https://i.postimg.cc/cC26Y72c/heavy-breathing-cat.png>)

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/artigos/food-porn/#comment_1114027202>):

> Uma vez eu vi um vídeo dessa tal de Boki da última imagem comendo ovos cozidos e cara, a mina é um saco sem fundo! Tipo, eu até sei que alguns desses *meokbangers* cospem/vomitam a comida depois de comer — todo "reality show" tem disso — mas se ela realmente comer tudo ela é tipo uma versão *live action* da Lendária Rainha Glutona 😝

Eu vi esse vídeo no TikTokerson. Na boa, foi nojento

**Comment by maryon-telvanni** [comment]
Imagens muito bem escolhidas. Cada uma parece ainda mais deliciosa que a outra

**Comment by angellisy** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/artigos/food-porn/#comment_1114029151>):

> Imagens muito bem escolhidas. Cada uma parece ainda mais deliciosa que a outra

Concordo plenamente

**Comment by najarin-the-loremaster** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/artigos/food-porn/#comment_1114029151>):

> Imagens muito bem escolhidas. Cada uma parece ainda mais deliciosa que a outra

Foram escolhidas a dedo em meio a muitas, muitas outras e, ainda assim, tratadas depois pra parecem ainda mais saborosas

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/artigos/food-porn/#comment_1114029637>):

> Concordo plenamente

Fiquei babando quando vi aquele lanche gigantesco com a carne brilhando :drooling\_face:
